examples/problem_decomposition/mifno.ipynb

```diff
@@ -299,35 +314,36 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "/home/alex/VirtEnvs/qsdk/lib/python3.10/site-packages/pyscf/dft/libxc.py:772: UserWarning: Since PySCF-2.3, B3LYP (and B3P86) are changed to the VWN-RPA variant, the same to the B3LYP functional in Gaussian and ORCA (issue 1480). To restore the VWN5 definition, you can put the setting \"B3LYP_WITH_VWN5 = True\" in pyscf_conf.py\n",
+      "/home/alex/.pyenv/versions/qsdk/lib/python3.10/site-packages/pyscf/dft/libxc.py:772: UserWarning: Since PySCF-2.3, B3LYP (and B3P86) are changed to the VWN-RPA variant, the same to the B3LYP functional in Gaussian and ORCA (issue 1480). To restore the VWN5 definition, you can put the setting \"B3LYP_WITH_VWN5 = True\" in pyscf_conf.py\n",
       "  warnings.warn('Since PySCF-2.3, B3LYP (and B3P86) are changed to the VWN-RPA variant, '\n"
      ]
     },
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Fragment (1,) mapped to 6 qubits.\n"
+      "Fragment (2,) mapped to 7 qubits.\n"
      ]
     }
    ],
    "source": [
     "from tangelo import SecondQuantizedMolecule\n",
     "from tangelo.toolboxes.operators import count_qubits\n",
     "from tangelo.toolboxes.qubit_mappings.mapping_transform import fermion_to_qubit_mapping\n",
+    "from tangelo.toolboxes.qubit_mappings import combinatorial\n",
     "\n",
     "# Needed to compute the molecular integrals.\n",
-    "mol = SecondQuantizedMolecule(BeH2_coordinates, q=charge, spin=spin, basis=basis)\n",
+    "mol = SecondQuantizedMolecule(xyz, q=charge, spin=spin, basis=basis)\n",
     "\n",
     "# Selection of a fragment and fetching the number of electrons and spinorbitals.\n",
-    "selected_fragment = \"(1,)\"\n",
+    "selected_fragment = \"(2,)\"\n",
     "n_electrons, n_spinorbitals = fno_fragments.n_electrons_spinorbs(selected_fragment)\n",
     "\n",
     "# Computing the related FermionOperator.\n",
     "ferm_op = fno_fragments.compute_fermionoperator(mol, selected_fragment)\n",
     "\n",
     "# Transformation of the FermionOperator to a QubitOperator.\n",
-    "qu_op = fermion_to_qubit_mapping(ferm_op, \"scBK\", n_spinorbitals, n_electrons, up_then_down=True)\n",
+    "qu_op = combinatorial(ferm_op, n_spinorbitals//2, n_electrons)\n",
```

Comment:
the combinatorial mapping is not well known and I'm not sure whether we are using it. I think you should mention that ahead of time in a sentence, and highlight that users could pick JW, etc so that they see things they are familiar with and understand.

if `fermion_to_qubit_mapping` is no longer used, please remove it

Reply:
I wanted to have the same systems for iFCI and MIFNO tests. However, iFCI does not work with smaller basis sets, so it required me to use `cc-pvdz`.

For this basis set, the number of qubits requires goes from 0 to 18 with no in-between. 18 qubits is too much to run in a single notebook. Therefore, this is why I chose to do a combinatorial mapping, as it maps to 7 qubits.